homeassistant/components/motion_blinds/strings.json

```diff
@@ -26,7 +26,7 @@
       "discovery_error": "Failed to discover a Motion Gateway"
     },
     "abort": {
-      "already_configured": "[%key:common::config_flow::abort::already_configured_device%], connection settings are updated",
```

Comment:
We can just use:

`[%key:common::config_flow::abort::already_configured_device%]`

As that is the default / common abort message we use across all integration.

Reply:
I changed it to only the string refrence.